docs/install/android.mdx

```diff
@@ -8,63 +8,81 @@ title: Build for Android
 
 In order to build PcapPlusPlus for Android please make sure you have the following prerequisites:
 
-1. A Linux machine - building on other platforms is currently not supported
-2. You should have [Android NDK](https://developer.android.com/ndk) installed
-3. Pre-compiled `libpcap` library for Android + header files which can be downloaded from [the `libpcap-android` GitHub repo](https://github.com/seladb/libpcap-android) (for Android API versions 21-30)
+1. [Android NDK](https://developer.android.com/ndk) should be installed
+2. Pre-compiled `libpcap` library for Android + header files which can be downloaded from [the `libpcap-android` GitHub repo](https://github.com/seladb/libpcap-android) (for Android API versions 21-30)
 
-## Configuration
+## Build
 
-Run the configuration script from PcapPlusPlus main directory:
+In order to build for Android a few parameters are needed:
 
-```bash
-./configure-android.sh
-```
+- The path of Android NDK. Let's assume it's stord in `ANDROID_NDK`
+- Android API version - must be between 21 and 30. Let's assume it's stord in `API_VERSIOM`
```

Comment:
same

Reply:
Fixed in https://github.com/PcapPlusPlus/pcapplusplus.github.io/pull/48/commits/cbb93cd88cf00bd702a3ec936eceba90f8bfc47f